src/services/supabase.ts

```diff
@@ -65,7 +68,16 @@ export enum FUNCTIONS {
 
 export const supabaseClient = createClient(
     supabaseSettings.url,
-    supabaseSettings.anonKey
+    supabaseSettings.anonKey,
+    {
+        // TODO (realtime) This is temporary until we figure out why some
+        //      subscriptions just hang forever.
+        realtime: {
+            logger: (kind: string, msg: string, data?: any) => {
+                LogRocket.log(kind, msg, data);
+            },
+        },
```

Comment:
This will make sure there are logs in LogRocket's UI for us to review. Hoping to remove this somewhat soon.